{
  "totalNumberSkippingTests": 17,
  "bugType": "passing_passing",
  "totalNumberErroringTests": 0,
  "repo": "google/error-prone",
  "metrics": {
    "FailureNames": [
      "java.lang.AssertionError"
    ],
    "StepsDurationsInSeconds": {
      "CheckoutBuggyBuildSourceCode": 0,
      "ComputeClasspath": 1,
      "ComputeTestDir": 0,
      "GatherTestInformationPreviousBuildSourceCode": 0,
      "ResolveDependency": 13,
      "BuildProjectBuild": 15,
      "BuildProjectPreviousBuildSourceCode": 27,
      "TestProjectBuild": 918,
      "ComputeSourceDir": 0,
      "InitRepoToPush": 6,
      "TestProjectPreviousBuildSourceCode": 655,
      "GatherTestInformationBuild": 0,
      "CloneRepository": 30,
      "CheckoutPatchedBuild": 0,
      "NopolRepair": 41,
      "PushIncriminatedBuild": 5,
      "CommitPatch": 6
    },
    "NbFailingTests": 13,
    "BugCommitUrl": "http://github.com/google/error-prone/commit/9e7e37d4320ac68419b13a6f2b329fcf7dce918d",
    "PatchCommitUrl": "http://github.com/google/error-prone/commit/06569874d5fd1b9c34f20c222e39e45e9dca5aa1",
    "ReproductionDate": "May 11, 2017 4:08:18 PM",
    "BuggyBuildId": 196717342,
    "PatchedBuilId": 199115933,
    "BuggyBuildURL": "http://travis-ci.org/google/error-prone/builds/196717342",
    "PatchedBuildURL": "http://travis-ci.org/google/error-prone/builds/199115933",
    "BuggyBuildDate": "Jan 30, 2017 9:38:55 PM",
    "PatchedBuildDate": "Feb 7, 2017 8:26:08 AM",
    "BugCommit": "9e7e37d4320ac68419b13a6f2b329fcf7dce918d",
    "PatchCommit": "06569874d5fd1b9c34f20c222e39e45e9dca5aa1",
    "PatchChangedFiles": 7,
    "PatchAddedLines": 0,
    "PatchDeletedLines": 0,
    "NbRunningTests": 1893,
    "NbLibraries": 52,
    "NbFileApp": 1026,
    "NbFileTests": 1420,
    "NbCPU": 56,
    "FreeMemory": 3945830728,
    "TotalMemory": 4709154816,
    "AngelicValuesByTest": {
      "com/google/errorprone/bugpatterns/JUnit4TestNotRunTest": 0,
      "com/google/errorprone/bugpatterns/ForOverrideCheckerTest": 0,
      "com/google/errorprone/bugpatterns/AmbiguousMethodReferenceTest": 0
    },
    "FreeMemoryByStep": {
      "CheckoutBuggyBuildSourceCode": 1225178560,
      "ComputeClasspath": 1587033936,
      "ComputeTestDir": 1268516960,
      "GatherTestInformationPreviousBuildSourceCode": 1082545664,
      "ResolveDependency": 1205282096,
      "BuildProjectBuild": 1375436752,
      "BuildProjectPreviousBuildSourceCode": 1186510680,
      "TestProjectBuild": 1359798896,
      "ComputeSourceDir": 1282957040,
      "InitRepoToPush": 1612527792,
      "TestProjectPreviousBuildSourceCode": 1126441752,
      "GatherTestInformationBuild": 1340689736,
      "CloneRepository": 1330606576,
      "CheckoutPatchedBuild": 1391074640,
      "NopolRepair": 1486619064,
      "PushIncriminatedBuild": 815964296,
      "CommitPatch": 3945830728
    }
  },
  "bugCommit": "https://github.com/google/error-prone/compare/3974a3cccc61...9e7e37d4320a",
  "failingModule": "/root/workspace/google/error-prone/196717342/core",
  "hostname": "spirals-spoon3r",
  "totalNumberFailingTests": 13,
  "error-types": [
    "java.lang.AssertionError"
  ],
  "totalNumberRunningTests": 1893,
  "failing-test-cases": [
    {
      "className": "com.google.errorprone.bugpatterns.AmbiguousMethodReferenceTest",
      "failingMethods": [
        "com.google.errorprone.bugpatterns.AmbiguousMethodReferenceTest#suppressedAtBothMethods",
        "com.google.errorprone.bugpatterns.AmbiguousMethodReferenceTest#suppressedAtMethod"
      ],
      "erroringMethods": [],
      "failures": [
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Saw unexpected error on line 7. All errors:\n[/A.java:7: warning: [AmbiguousMethodReference] This method\u0027s reference is ambiguous, its name and functional interface type are the same as: c(A, D)\n  B c(D d) {\n    ^\n    (see http://errorprone.info/bugpattern/AmbiguousMethodReference)]",
          "isError": false
        },
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Saw unexpected error on line 7. All errors:\n[/A.java:7: warning: [AmbiguousMethodReference] This method\u0027s reference is ambiguous, its name and functional interface type are the same as: c(A, D)\n  B c(D d) {\n    ^\n    (see http://errorprone.info/bugpattern/AmbiguousMethodReference)]",
          "isError": false
        }
      ],
      "nbFailures": 2,
      "nbErrors": 0
    },
    {
      "className": "com.google.errorprone.bugpatterns.ForOverrideCheckerTest",
      "failingMethods": [
        "com.google.errorprone.bugpatterns.ForOverrideCheckerTest#testUserCanCallSuperFromOverridden"
      ],
      "erroringMethods": [],
      "failures": [
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Saw unexpected error on line 5. All errors:\n[/test/Test.java:5: error: [ForOverride] Method annotated @ForOverride must not be invoked directly (except by the declaring class, test.ExtendMe)\n    return super.overrideMe();\n                           ^\n    (see http://errorprone.info/bugpattern/ForOverride)]",
          "isError": false
        }
      ],
      "nbFailures": 1,
      "nbErrors": 0
    },
    {
      "className": "com.google.errorprone.bugpatterns.JUnit4TestNotRunTest",
      "failingMethods": [
        "com.google.errorprone.bugpatterns.JUnit4TestNotRunTest#testContainsExpectAsIdentifier",
        "com.google.errorprone.bugpatterns.JUnit4TestNotRunTest#testContainsVerifyAsIdentifier",
        "com.google.errorprone.bugpatterns.JUnit4TestNotRunTest#testContainsQualifiedVerify",
        "com.google.errorprone.bugpatterns.JUnit4TestNotRunTest#testContainsAssertAsIdentifier",
        "com.google.errorprone.bugpatterns.JUnit4TestNotRunTest#testContainsQualifiedAssert",
        "com.google.errorprone.bugpatterns.JUnit4TestNotRunTest#testContainsQualifiedCheck",
        "com.google.errorprone.bugpatterns.JUnit4TestNotRunTest#testContainsCheckAsIdentifier",
        "com.google.errorprone.bugpatterns.JUnit4TestNotRunTest#testContainsQualifiedExpect",
        "com.google.errorprone.bugpatterns.JUnit4TestNotRunTest#testContainsFailAsIdentifier",
        "com.google.errorprone.bugpatterns.JUnit4TestNotRunTest#testContainsQualifiedFail"
      ],
      "erroringMethods": [],
      "failures": [
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Did not see an error on line 7 matching @Test. All errors:\n[]",
          "isError": false
        },
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Did not see an error on line 7 matching @Test. All errors:\n[]",
          "isError": false
        },
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Did not see an error on line 7 matching @Test. All errors:\n[]",
          "isError": false
        },
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Did not see an error on line 7 matching @Test. All errors:\n[]",
          "isError": false
        },
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Did not see an error on line 7 matching @Test. All errors:\n[]",
          "isError": false
        },
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Did not see an error on line 7 matching @Test. All errors:\n[]",
          "isError": false
        },
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Did not see an error on line 7 matching @Test. All errors:\n[]",
          "isError": false
        },
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Did not see an error on line 7 matching @Test. All errors:\n[]",
          "isError": false
        },
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Did not see an error on line 8 matching @Test. All errors:\n[]",
          "isError": false
        },
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Did not see an error on line 7 matching @Test. All errors:\n[]",
          "isError": false
        }
      ],
      "nbFailures": 10,
      "nbErrors": 0
    }
  ]
}